great-idea-website/css/index.css

```diff
@@ -65,66 +65,107 @@ h1, h2, h3, h4, h5 {
 
 /* Your code starts here! */
 
-main {
+.main-content-area {
 	width: 900px;
 	display: block;
 	margin: auto;
+	margin-top: 20px;
 }
-header {
-	height: 80px;
-	padding: 5px;
+
+/* Main Navigation Styling */
+
+.main-nav {
 	margin: auto;
-	vertical-align: middle;
+	/*text-align: center;*/
+	width: 900px;
+	height: 100px;
+	position: relative;
 }
-nav {
+
+nav div {
 	display: inline-block;
+	/*height: 60px;*/
+	margin: 25px 50px 25px 0px;
 }
+
 nav a {
+	display: inline-block;
+	padding: 30px;
 	text-decoration: none;
 	color: gray;
 	font-size: 15px;
-	font-family: Helvetica;
-	width: 60px;
-	height: 20px;
-	padding: 20px;
+
+}
+
+nav a:first-child {
+	padding-left: 0;
 }
+
 .logo {
-	display: inline-block;
-	padding: 20px;
-	margin: 0px 0px 0px 50px;
+	position: absolute;
```

Comment:
if you do a position : absolute here, you will need to keep position : relative on the <nav class=main-nav> element.  And then you can adjust the position using the top / bottom / left / right properties.  Alternatively, you can avoid all that, and can simply do a position : relative here, take out the position : absolute on the nav, and then tweak the position using top / bottom / left / right as well.  

position : relative is tweaked from it's normal place in the document flow.  position : absolute is confusingly positionred relative to any ancestor element that has a position specified (aside from static).  if there are no "positioned" elements, then the position absolute element will be relative to the body tag.